scripts/publish_benchmark_result.py

```diff
@@ -0,0 +1,414 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import datetime
+import json
+import os
+import pathlib
+from pathlib import Path
+import subprocess
+import sys
+from typing import Dict, List, Optional, Tuple, Union
+
+import pandas as pd
+import pandas_gbq
+
+LOGGING_NAME_ENV_VAR = "BIGFRAMES_PERFORMANCE_LOG_NAME"
+CURRENT_DIRECTORY = pathlib.Path(__file__).parent.absolute()
+
+
+def run_benchmark(base_path):
+    # Run benchmarks in parallel session.run's, since each benchmark
+    # takes an environment variable for performance logging
+    processes = process_benchmark_recursively(base_path)
+    for process in processes:
+        process.wait()
```

Comment:
What happens if the benchmark fails?

Per https://docs.python.org/3/library/subprocess.html#subprocess.Popen.wait it sets the returncode attribute, but that doesn't turn into an exception or failure return code in this script.

Idea: Add up all return codes and return that from `run_benchmark`

Reply:
Updated to not use subprocess to capture issue.